pysand/erosion.py

```diff
@@ -196,12 +229,8 @@ def manifold(v_m, rho_m, mu_m, Q_s, GF, D, d_p, Dm):
     :return: Manifold erosion rate [mm/year]
     '''
 
-    # Unique input validation
-    if not isinstance(Dm, (int, float)):
-        raise exc.FunctionInputFail('{} is not a number'.format(Dm))
-
-    # Common input validation
-    kwargs = {'v_m': v_m, 'rho_m': rho_m, 'mu_m': mu_m, 'Q_s': Q_s, 'GF': GF, 'D': D, 'd_p': d_p}
+    # Input validation
+    kwargs = {'v_m': v_m, 'rho_m': rho_m, 'mu_m': mu_m, 'Q_s': Q_s, 'GF': GF, 'D': D, 'd_p': d_p, 'Dm': D}
```

Comment:
Hi, it seems like you are using the `D` input two times, for both `D` and `Dm`. I think the last part of the line should be `'Dm':Dm`, agree?

Reply:
Hi, yes agree

Reply:
sounds good. Otherwise good work on the rest :)